scripts/auth.js

```diff
@@ -2,15 +2,13 @@
 function onSignIn(googleUser) {
     // Useful data for your client-side scripts:
     const profile = googleUser.getBasicProfile();
-    console.log("ID: " + profile.getId()); // Don't send this directly to server!
-    console.log('Full Name: ' + profile.getName());
-    console.log("Email: " + profile.getEmail());
-
-    // The ID token you need to pass to your backend:
-    const id_token = googleUser.getAuthResponse().id_token;
-    console.log("ID Token: " + id_token);
+    const name = profile.getName();
+    const email = profile.getEmail();
+    //To send the information to the server
+    const request = new Request("/addUser?name=" + name + "&email=" + email, {method: "POST"}); 
```

Comment:
You can move this to body. Ideally we should consume query parameters to limit your query in GET. Also there is limit of query parameter size which you don't have on request body.